### PR TITLE
Employee enters review period on report

### PR DESCRIPTION
**Pull request by robmckinnon**
- Add review_period to reports.
- User can edit period on report edit view.

### CODE REVIEW

**Comment by novotnyjakub** [comment]
The new `review_period` is not required?

File: db/schema.rb [hunk 1]
```diff
@@ -61,11 +62,19 @@
...
     t.string   "grade"
     t.string   "organisation"
+    t.string   "budget_type_1"
+    t.string   "budget_type_2"
+    t.string   "budget_value_1"
+    t.string   "budget_value_2"
+    t.string   "budget_description_1"
+    t.string   "budget_description_2"
+    t.string   "staff_count_grade"
+    t.string   "staff_engagement_score"
```
Comment (novotnyjakub) [inline db/schema.rb]: Am I missing something, or there's no migration for all these fields?

**Comment by robmckinnon** [comment]
In general we don't have validations for the presence of any fields, `review_period` is following that convention.

File: db/schema.rb (hunk 1 above)
Comment (robmckinnon) [inline db/schema.rb]: Well spotted. I'll remove these.

**Comment by novotnyjakub** [comment]
:+1: no problem

**Pull request merged**